user_docs/en/changes.t2t

```diff
@@ -9,9 +9,12 @@ What's New in NVDA
 - In Windows 11, NVDA will announce alerts from voice typing and suggested actions including the top suggestion when copying data such as phone numbers to the clipboard (Windows 11 2022 Update and later). (#16009, @josephsl)
 -
 
-
 == Changes ==
 
+- Add-on Store:
+  - The minimum NVDA version and the last tested version of NVDA for an add-on is now displayed in the "other details" area. (#15776, @Nael-Sayegh)

```

Comment:
To be more precise and grammatically correct:

```suggestion
  - The minimum NVDA version and the last tested NVDA version of NVDA for an add-on are now displayed in the "other details" area. (#15776, @Nael-Sayegh)
```

or shorter with "factorisation" of "of NVDA":

```suggestion
  - The minimum and the last tested NVDA versions of NVDA for an add-on are now displayed in the "other details" area. (#15776, @Nael-Sayegh)
```